src/main/java/bio/terra/cli/businessobject/Group.java

```diff
@@ -17,7 +17,6 @@
  */
 public class Group {
   private static final Logger logger = LoggerFactory.getLogger(Group.class);
```

Comment:
I prefer having a blank line between logger and other variables. That's what most of our code does. Can these be reverted?